### ISSUE TITLE
Rune buy price doesn't match

### ISSUE THREAD

**Issue by Blackdead** [gwdevhub/GWToolboxpp#1213]
Hey,
the buy info from toolbox doesn't match with with the real one form the rune trader. 700g !=110g.
I haven't seen this before the recent GW updates. Maybe it's due to the update?
![grafik](https://github.com/user-attachments/assets/21886a77-d397-4388-b25c-17688fb884a5)
![grafik](https://github.com/user-attachments/assets/a59440bc-fa54-49d2-b0b7-fada4aadfd65)

**Comment by 3vcloud** [comment]
How much to buy?

**Comment by Blackdead** [comment]
was 110g. sorry had the wrong one picked.

**Comment by Blackdead** [comment]
I checked the code and it's seems that it is not a specific problem within the toolbox application but within kamand trading chat part/website. The JSON from https://kamadan.gwtoolbox.com/trader_quotes for Minor Command for e. g. is:
`"15548-25B80000240801852530030B21E82601":{"p":700,"t":"1716856069"},`
If I interpreted the data right "t" is the timestamp at which the information is updated and that timestamp is Tue May 28 2024 00:27:49 GMT+0000 which is outdated. This probably affects all others as well.

I guess this issue should be moved to the kamadan-trading-chat repo.
